qmlcomps/videocamera/Groove.qml

```diff
@@ -0,0 +1,25 @@
+import QtQuick 2.0
```

Comment:
2.0?

Reply:
it was already in the file i take directly from original repo, that as no effect in the app